non-oss/addons/kontena-lens/resources/07-dashboard-deployment.yml

```diff
@@ -0,0 +1,59 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: dashboard
+  namespace: kontena-lens
+  labels:
+    app: dashboard
+spec:
+  selector:
+    matchLabels:
+      app: dashboard
+  template:
+    metadata:
+      labels:
+        app: dashboard
+    spec:
+      serviceAccountName: lens-operator
+      restartPolicy: Always
+      containers:
+        - image: quay.io/kontena/lens:latest
+          name: dashboard
+          imagePullPolicy: Always
+          env:
+            - name: KUBE_KONTENA_URL
+              value: http://usermanagement:9999
+            - name: KUBE_TERMINAL_URL
+              value: http://localhost:9998
+            - name: REDIS_CLIENT_HOST
+              value: redis
```

Comment:
I think we should enable built-in user management by default or add it behind config variable:
```
- name: USER_MANAGEMENT_ENABLED
   value: "true"
```